client/src/app/pages/applications/applications-table-assessment/applications-table-assessment.tsx

```diff
@@ -445,9 +451,11 @@ export const ApplicationsTable: React.FC = () => {
     if (applicationWriteAccess) {
       actions.push({
         title: t("actions.delete"),
-        ...(row.migrationWave !== null && {
+        ...((row.migrationWave !== null || ticketByApplication(row.id)) && {
```

Comment:
Is this fix exclusive to the assessments page?

Reply:
Yeah it's needed for analyze table too, I forgot, thanks

Reply:
Np. Also after testing - The tooltip is showing up for the wrong application in the table. Something is wrong with the association logic for tickets <-> applications.